libs/api/domains/cms/src/lib/cms.elasticsearch.service.ts

```diff
@@ -41,13 +41,21 @@ export class CmsElasticsearchService {
 
   async getArticles(
     index: string,
-    { category, size }: GetArticlesInput,
+    input: GetArticlesInput,
   ): Promise<Article[]> {
     const query = {
       types: ['webArticle'],
-      tags: [{ type: 'category', key: category }],
+      tags: [],
       sort: { 'title.sort': 'asc' as sortDirection },
-      size,
+      size: input.size,
+    }
+
+    if (input.category) {
+      query.tags.push({ type: 'category', key: input.category })
+    }
+
+    if (input.organization) {
+      query.tags.push({ type: 'organization', key: input.organization })
     }
```

Comment:
This might not work as expected.
I believe the query underlying `getDocumentsByMetaData` might not scope tags as expected.
Meaning a query that requests a document by `category` and `organization` tag will return documents having either a matching category tag or matching organization tag.
While I think the expected behavior is returning documents that have both tags.

If you require this kind of scoping maybe we should look into changing the  underlying query to allow for this?

Reply:
It seems to work as expected. I did some playing around and the following query returns 6 articles
```
{
  getArticles(input:{lang: "is", organization: "syslumenn", category: "akstur-og-bifreidar"}) {
    title
  }
}
```

But if I exclude the category parameter I get all 54 articles belonging to Sýslumenn.

Reply:
Nice, never been so happy to be wrong.
I'll approve